controllers/snapshot/snapshot_controller_test.go

```diff
@@ -140,6 +140,12 @@ var _ = Describe("SnapshotController", func() {
 	})
 
 	It("can Reconcile function prepare the adapter and return the result of the reconcile handling operation", func() {
+		req := ctrl.Request{
```

Comment:
If we set the `req` to a non-existing CR and remove `Expect(setupCache(k8sManager)).To(Succeed())` and some cache functions, the case can pass. But some code will not be covered ([coverage report](https://app.codecov.io/gh/redhat-appstudio/integration-service/pull/225/blob/controllers/snapshot/snapshot_controller.go)), release-service coverage [here](https://app.codecov.io/gh/redhat-appstudio/release-service/blob/main/controllers%2Frelease%2Fcontroller.go#L79) is similar.

If we use a non-existing CR and remove `Expect(setupCache(k8sManager)).To(Succeed())` from other _suite_test.go, they are similar.

Can we do like this? Thanks!

Reply:
Yes, I think that your proposed change is reasonable.